File: admin/src/main/java/com/fahamutech/adminapp/database/connector/HomeDataSource.java
package com.fahamutech.adminapp.database.connector;

import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
import androidx.recyclerview.widget.RecyclerView;

public interface HomeDataSource {

    Object getCategory(RecyclerView recyclerView, SwipeRefreshLayout swipeRefreshLayout);

    Object getTestimony(RecyclerView recyclerView, SwipeRefreshLayout swipeRefreshLayout);

}

File: core/src/main/java/com/fahamutech/kcore/user/database/connector/HomeDataSource.kt
package com.fahamutech.kcore.user.database.connector;

import androidx.recyclerview.widget.RecyclerView;
import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;

public interface HomeDataSource {

    void getCategory(RecyclerView recyclerView, SwipeRefreshLayout swipeRefreshLayout);

    void getTestimony(RecyclerView recyclerView, SwipeRefreshLayout swipeRefreshLayout);
}
